no-browser-site-scrapers/WhittierHealth/index.js

```diff
@@ -0,0 +1,264 @@
+const { site } = require("./config");
+const fetch = require("node-fetch");
+const htmlParser = require("node-html-parser");
+const moment = require("moment");
+
+module.exports = async function GetAvailableAppointments(
+    _ignored,
+    fetchService = liveFetchService()
+) {
+    console.log(`${site.name} starting.`);
+
+    const websiteData = await ScrapeWebsiteData(site, fetchService);
+
+    const results = {
+        parentLocationName: `${site.name}`,
+        timestamp: moment().format(),
+        individualLocationData: [websiteData],
+    };
+
+    console.log(`${site.name} done.`);
+
+    return results;
+};
+
+/**
+ * Dependency injection: in live scraping, the fetchAvailability() in this module is used.
+ * In testing, a mock fetchAvailability() is injected.
+ */
+function liveFetchService() {
+    return {
+        async fetchFrontPage(frontPageUrl) {
+            return await fetchFrontPage(frontPageUrl);
+        },
+        async fetchCalendarPage(calendarUrl) {
+            return await fetchCalendarPage(calendarUrl);
+        },
+    };
+}
+
+async function ScrapeWebsiteData(site, fetchService) {
+    const availabilityContainer = {
+        availability: {},
+        hasAvailability: false,
+    };
+
+    const linksArray = await getCalendarLinks(fetchService, site.signUpLink);
+
+    let hasAvailability = false;
+
+    // Advance the calendar of each site until no availability is found.
+    await Promise.all([
+        linksArray.forEach(async (link) => {
+            console.log(`calendarPageLink: ${link}`);
+
+            const monthAvailabilityMap = await getSlotsFromPage(
+                fetchService,
+                link
+            );
+
+            // Add all day objects to results.availability
+            monthAvailabilityMap.forEach((value, key) => {
+                availabilityContainer.availability[key] = {
+                    numberAvailableAppointments: value,
+                    hasAvailability: !!value,
+                };
+            });
+
+            hasAvailability ||=
+                Object.keys(availabilityContainer.availability).length > 0;
```

Comment:
I am getting an "invalid token '||='" error when attempting to run this.  I'm running node.js v14.15.5.  According to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment), this construct isn't fully supported until node.js v15. (which we are requiring yet).

package.json sets the minimum requirement at `        "node": ">= 14.0.0",`

Reply:
Ok, I'll fix that. Also see about downgrading to 14.x

Reply:
Committed and downgrade done.